CMP1903_A1_2324/Game.cs

```diff
@@ -1,21 +1,51 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using DieGame_Assessment_1;
+using System;
+using System.Collections;
+using System.Diagnostics;
 
-namespace CMP1903_A1_2324
+namespace DieGame_Assessment_1
 {
-    internal class Game
+    // Game class
+    public class Game
     {
-        /*
-         * The Game class should create three die objects, roll them, sum and report the total of the three dice rolls.
-         *
-         * EXTRA: For extra requirements (these aren't required though), the dice rolls could be managed so that the
-         * rolls could be continous, and the totals and other statistics could be summarised for example.
-         */
+        // Stores the random object for number generation
+        private Random rng;
+        // Creates the three public die objects
+        public Die die1;
+        public Die die2;
+        public Die die3;
 
-        //Methods
+        // Game class constructor
+        public Game()
+        {
+            // Initliaze the random object
+            rng = new Random();
```

Comment:
While not incorrect, it would be good practice for all of these variables to have a specified type (so it's easier to read and enforce).
`Random rng = new  Random();`, `Die die1 = new Die(rng);`, etc.